config/crd/bases/infrastructure.cluster.x-k8s.io_awsmachinetemplates.yaml

```diff
@@ -397,13 +398,17 @@ spec:
                             type: integer
                           size:
                             description: Size specifies size (in Gi) of the root storage
-                              device
+                              device. Must be greater than the image root snapshot
+                              size or 8 (whichever is greater).
                             format: int64
+                            minimum: 8
```

Comment:
I know we talked about 10 here @vincepri but 8 seemed like a more sensible guard given that it's the default.